cool-core/src/test/java/com/nus/cool/core/io/storevector/ZIntBitInputVectorTest.java

```diff
@@ -0,0 +1,54 @@
+package com.nus.cool.core.io.storevector;
+
+import com.google.common.collect.Maps;
+import com.nus.cool.core.io.DataOutputBuffer;
+import com.nus.cool.core.io.compression.Compressor;
+import com.nus.cool.core.io.compression.Histogram;
+import com.nus.cool.core.io.compression.ZIntBitCompressor;
+import com.nus.cool.core.io.storevector.ZIntBitInputVector;
+import com.nus.cool.core.schema.CompressType;
+import com.nus.cool.core.util.ArrayUtil;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+public class ZIntBitInputVectorTest{
+    @Test(dataProvider = "ZIntBitDP")
```

Comment:
Change the UnitTest Method to `ZIntBitInputVectorUnitTest` not `testRead`

Reply:
Thanks, will update.